app/(dashboard)/mailboxes/[mailbox_slug]/[category]/conversation/messageActions.tsx

```
@@ -402,6 +404,16 @@ export const MessageActions = () => {
             </>
           ))}
       </div>
+      <div className="flex items-center gap-2 mt-2">
+        <Button
+          variant="ghost"
+          size="sm"
+          onClick={onToggleCc}
+          className="text-muted-foreground hover:text-foreground"
+        >
```

ESLint

[error] Replace `⏎··········variant="ghost"⏎··········size="sm"⏎··········onClick={onToggleCc}⏎··········className="text-muted-foreground·hover:text-foreground"⏎········` with `·variant="ghost"·size="sm"·onClick={onToggleCc}·className="text-muted-foreground·hover:text-foreground"`

(prettier/prettier)